src/app/components/demo.component.ts

```diff
@@ -41,19 +41,16 @@ let PreviewID = 0;
   template: `
     <ng-content></ng-content>
 
-    <div style="display:flex; align-items: flex-start; justify-content: space-between;">
-      <div *ngFor="let tab of tabs" style="{{ tabs.length > 1 ? 'width: 49.5%' : 'width: 100%' }}">
-        <h5>{{ tab.name }}</h5>
+    <div cds-layout="grid gap:md">
```

Comment:
Thanks for cleaning this up, we could also remove the `embedStackblitz` method.

Reply:
Done